unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Materials/Tests/PBRMaterialVisualTests.cs

```diff
@@ -9,16 +11,12 @@
 
 public class PBRMaterialVisualTests : VisualTestsBase
 {
-    [UnityTest]
-    [VisualTest]
-    [Explicit]
-    [Category("Explicit")]
+    [UnityTest, VisualTest]
+    [Explicit, Category("Explicit")]
```

Comment:
should we still keep these as explicit?

Reply:
These are legacy tests, it was not the goal of this PR to assert the state of the old ones.

Reply:
These are legacy tests, it was not the goal of this PR to assert the state of the old ones.